lib/mpl_toolkits/basemap/test.py

```diff
@@ -190,8 +193,74 @@ def test_results_should_be_same_for_c_and_f_order_arrays(self):
         assert_almost_equal(xx1, xx2)
         assert_almost_equal(yy1, yy2)
 
-
-
+@skipIf(LooseVersion(np.__version__) < LooseVersion("1.10.0"), 
+        "Test skipped if NumPy version is less than 1.10.x")
+class TestLSMask(Basemap, unittest.TestCase):
+	def make_map_object(self,**kwargs):
+		""" Create a Basemap class instance """
+		if PY3:
+			# If Python 3, use new super() syntax
+			m = super().__init__(**kwargs)
+		else:
+			# Python 2 super syntax
+			m = super(TestLSMask,self).__init__(**kwargs)
+		return m
```

Comment:
I am not sure why you have this method. Why not just make a regular basemap object?